src/utils/holoflowKits.ts

```diff
@@ -0,0 +1,12 @@
+import { LiveSelector, MutationObserverWatcher } from '@holoflows/kit'
+
+export const untilElementAvailable: {
+    (ls: LiveSelector<HTMLElement, true>): Promise<any>
+    (ls: LiveSelector<HTMLElement, false>): Promise<any>
```

Comment:
You can write `(ls: LiveSelector<HTMLElement, boolean>): Promise<void>` instead of two function overload.

Use void instead of any here

Reply:
got it.